bash-hash/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "bash-hash"
+version = "0.1.0"
+description = "bash hash function (STB 34.101.77-2020)"
+authors = ["RustCrypto Developers"]
+license = "MIT OR Apache-2.0"
+readme = "README.md"
+edition = "2024"
+rust-version = "1.85"
+documentation = "https://docs.rs/belt-hash"
+repository = "https://github.com/RustCrypto/hashes"
+keywords = ["belt", "stb", "hash", "digest"]
+categories = ["cryptography", "no-std"]
+
+[dependencies]
+digest = "0.11.0-rc.3"
+bash-f = { version = "0.1.0-rc.0" }
+subtle = { version = "2", default-features = false }
```

Comment:
IIUC `subtle` is not used anywhere in this crate.

Reply:
Youre right, forgot to remove it